.github/compatibility-test-matrices/release-v5.0.x.json

```diff
@@ -1,12 +1,485 @@
+      "chain-b-tag": "release-v5.0.x",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestIncentivizedTransferTestSuite",
+      "test": "TestPayPacketFeeAsync_SingleSender_NoCounterPartyAddress"
+    },
+
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "release-v5.0.x",
+      "chain-b-tag": "release-v5.0.x",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestIncentivizedTransferTestSuite",
+      "test": "TestMsgPayPacketFee_AsyncMultipleSenders_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "release-v5.0.x",
+      "chain-b-tag": "v4.0.0",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestIncentivizedTransferTestSuite",
+      "test": "TestMsgPayPacketFee_AsyncMultipleSenders_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "v4.0.0",
+      "chain-b-tag": "release-v5.0.x",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestIncentivizedTransferTestSuite",
+      "test": "TestMsgPayPacketFee_AsyncMultipleSenders_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "release-v5.0.x",
+      "chain-b-tag": "release-v5.0.x",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestClientTestSuite",
+      "test": "TestClientUpdateProposal_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "release-v5.0.x",
+      "chain-b-tag": "v4.0.0",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestClientTestSuite",
+      "test": "TestClientUpdateProposal_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "release-v5.0.x",
+      "chain-b-tag": "v3.0.0",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestClientTestSuite",
+      "test": "TestClientUpdateProposal_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "release-v5.0.x",
+      "chain-b-tag": "v2.0.0",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestClientTestSuite",
+      "test": "TestClientUpdateProposal_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "v4.0.0",
+      "chain-b-tag": "release-v5.0.x",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestClientTestSuite",
+      "test": "TestClientUpdateProposal_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "v3.0.0",
+      "chain-b-tag": "release-v5.0.x",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestClientTestSuite",
+      "test": "TestClientUpdateProposal_Succeeds"
+    },
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "v2.0.0",
+      "chain-b-tag": "release-v5.0.x",
+      "chain-image": "ghcr.io/cosmos/ibc-go-simd",
+      "entrypoint": "TestClientTestSuite",
+      "test": "TestClientUpdateProposal_Succeeds"
```

Comment:
these 2 tests are failing, is this expected?

Reply:
sync'd with @charleenfei and it looks like the `TrustingPeriod` field was backported to 2.4 and 3.2 (not 2.0/3.0!)